tests/autologging/test_autologging_utils.py

```diff
@@ -396,3 +394,97 @@ def test_batch_metrics_logger_continues_if_log_batch_fails(
         assert metric.key == "y"
         assert metric.value == 2
         assert metric.step == 1
+
+
+def test_autologging_integration_calls_underlying_function_correctly():
+    @autologging_integration("test_integration")
+    def autolog(foo=7, disable=False):
+        return foo
+
+    assert autolog(foo=10) == 10
+
+
+def test_autologging_integration_stores_and_updates_config():
+    @autologging_integration("test_integration")
+    def autolog(foo=7, bar=10, disable=False):
+        return foo
+
+    autolog()
+    assert AUTOLOGGING_INTEGRATIONS["test_integration"] == {"foo": 7, "bar": 10, "disable": False}
+    autolog(bar=11)
+    assert AUTOLOGGING_INTEGRATIONS["test_integration"] == {"foo": 7, "bar": 11, "disable": False}
+    autolog(foo=6, disable=True)
+    assert AUTOLOGGING_INTEGRATIONS["test_integration"] == {"foo": 6, "bar": 10, "disable": True}
+
+
+def test_autologging_integration_validates_structure_of_autolog_function():
+    def fn_missing_disable_conf():
+        pass
+
+    def fn_bad_disable_conf_1(disable=True):
+        pass
+
+    # Try to use a falsy value that isn't "false"
+    def fn_bad_disable_conf_2(disable=0):
+        pass
+
+    for fn in [fn_missing_disable_conf, fn_bad_disable_conf_1, fn_bad_disable_conf_2]:
+        with pytest.raises(Exception) as exc:
+            autologging_integration("test")(fn)
+        assert "must specify a 'disable' argument" in str(exc)
```

Comment:
```suggestion
        with pytest.raises(Exception, match="must specify a 'disable' argument") as exc:
            autologging_integration("test")(fn)
```

Shoule we use `match` argument here?

Reply:
Absolutely! I've replaced all instances of `... in str(exc)` with match; thanks so much for the suggestion!